### PR TITLE
Team updates

### PR DESCRIPTION
**Pull request by dpordomingo**
some changes in team:

Promotions: 
- Victor

New people:
- Alex Dahlin
- Guillem
- Tristan
- Robbie
- Hassen

People that quitted:
- Francesc

Already in staging
https://landing-staging.srcd.run/company/#team

team

### CODE REVIEW

File: hugo/data/teams.yml [hunk 1]
```diff
@@ -126,6 +130,10 @@ sections:
           name: Jan Hůla
           position: Machine Learning Intern
           github: https://github.com/Jan21
+        - picture: placeholder/male.png
+          name: Tristan Kalos
+          position: Machine Learning Intern
+          github: https://github.com/glimow
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: @vmarkovtsev is that one https://github.com/glimow our Tristan Kalos ?

File: hugo/data/teams.yml [hunk 2]
```diff
@@ -144,23 +152,29 @@ sections:
         github: https://github.com/gomesfernanda
 
 
-  - name: Developer Relations
+  - name: Marketing
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: @vcoisne is this one the name of the new team?
Should we change its headline?

File: hugo/data/teams.yml [hunk 3]
```diff
@@ -144,23 +152,29 @@ sections:
...
+        name: Alex Dahlin
+        position: Marketing Intern
+        linkedin: https://www.linkedin.com/in/alexanderdahlin/
 
 
   - name: Operations
     headline: Taking care of the team at source{d}
+    members:
+      - picture: placeholder/male.png
+        name: Robbie Blue
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: @eiso I saw in our organization chart that @bluer73 reports to you; Anna was in Operations team, so I added him here; should we move to a different team?

File: hugo/data/teams.yml (hunk 1 above)
Comment (vmarkovtsev) [reply hugo/data/teams.yml]: IDK, he did not share his GitHub with me. Looks very truthful.

Comment (dpordomingo) [reply hugo/data/teams.yml]: Last question; when is he joining us @vmarkovtsev ?

Comment (vmarkovtsev) [reply hugo/data/teams.yml]: @esther knows, I don't...

**Comment by dpordomingo** [comment]
merged and deployed.

Feel free to request a change if the following is not correct and/or should be updated:
- New team **Sales** was created with headline &mdash;as proposed by Hassen&mdash;:
  >_"Revenue generation through strategic alliances with customers"_    
- **Hassen** was added into Sales as Head of Sales US,
- **Robbie** was added into Operations as Talent Manager,
- **Victor** was moved from DevRel to Management as VP of Marketing.

**Pull request merged**